tests/system/conftest.py

```diff
@@ -104,7 +104,12 @@ def delete_entity(ds_key):
 
 
 @pytest.fixture
-def client_context():
-    client = ndb.Client()
+def namespace():
+    return ""
```

Comment:
I was thinking if we replaced this with ``return str(uuid.uuid4())`` or other random string, this would fix our system test breakage by running each test in a different namespace. Would that screw up your metadata tests?

Reply:
I'll give it a try before merging.

Reply:
The uuid namespace breaks half the tests, but I think this is worth pursuing. I'll merge this and try that in another branch.